api/src/api.ts

```diff
@@ -52,6 +54,10 @@ async function startApp() {
   ui.use(serve(resolve(__dirname, './ui')));
   app.use(mount('/', ui));
 
+  app.on('error', (err, ctx) => {
```

Comment:
Do we need this?

Reply:
It is more of a safeguard when something on our `docker compose` goes wrong silently. I used these logs to see why my connection with redis and rabbit was broken sometimes.

Reply:
Forget it, it did not help when I tried to inspect it again, so I'm removing it.